tests/API/XCCDF/unittests/test_xccdf_oval_absolute_path.sh

```diff
@@ -56,9 +73,17 @@ for prefix in "" "file://";do
 
     cp "$ORIGINAL_XCCDF" "$srcdir/test_default_selector.oval.xml" "$testDir"
     cp "$srcdir/oval/always-fail/oval.xml" "$testDir/oval/always-fail"
-    XCCDF="$(readlink -e "$testDir/${name}.xccdf.xml")"
 
-    sed -i "s|%%TEST_DIR_PATH%%|${prefix}${testDir}|" $XCCDF
+    case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?